### PR TITLE
Some Utilities

### PR DESCRIPTION
**Pull request by Justinfungi**
1 r-precision: for reproducing the testing part of the dream fusion. Here is some of our trial before, but we only train a few epochs and haven't used the DMTet. The number can be larger with larger epochs and the use of DMTet. It is for testing
image

2: The prompt and the 3D mesh editing are part of my thought to separate the prompt into key items and put them into the same place. Kind of divide and conquer for the longer prompt ==> it may be a bit naive regarding the implementation of 3D mesh editing. This part is just for fun and to try the break the limit of dream fusion

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,16 +15,10 @@ trial*/
...
-data2
-data/car*
-data/chair*
-data/warrior*
-data/wd*
-data/space*
-data/corgi*
-data/turtle*
\ No newline at end of file
+<<<<<<< Updated upstream
```
Comment (ashawkey) [inline .gitignore:22]: I think these are wrongly added? Please keep the original ignore file.

File: Improvement/Pipeline.png
Comment (ashawkey) [inline Improvement/Pipeline.png:1]: what is this?

File: Improvement/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+### Improvement:
```
Comment (ashawkey) [inline Improvement/readme.md:1]: is this used or not used? I would prefer to only keep useful information...

**Comment by ashawkey** [comment]
Hi, could you check these reviews? Also, it's better to change the folder name `Improvement` to `evaluation`.

File: Improvement/Pipeline.png
Comment (Justinfungi) [reply Improvement/Pipeline.png:1]: This is for the 2 file: prompt.py and mesh_to_video.py

this picture present my idea that we can extract keywords from the long prompt => and then build a relationship mapping between the keywords. A relationship such as locational information ==> mesh_to_video.py is to manipulate the meshes and get a final output like this - combination of 2 output from dreamfusion. I currently figure out the prompt and the mesh part, but relationship mapping is not finished yet

**Comment by Justinfungi** [comment]
> Hi, could you check these reviews? Also, it's better to change the folder name `Improvement` to `evaluation`.

I have changed the folder name. do you think that I should keep the prompt.py and mesh_to_video.py or not? i propose it since i have thoughts like these. but it may be not appealing for you to add them to the project.

**Comment by ashawkey** [comment]
It's OK to keep them. Could you change the name to lower case `evaluation` as we name all folders that way? Also, is the `Pipeline.png` something necessary?

**Comment by Justinfungi** [comment]
> It's OK to keep them. Could you change the name to lower case `evaluation` as we name all folders that way? Also, is the `Pipeline.png` something necessary?

@ashawkey Nah, I think the Pipeline.png is for illustrating the idea. i have deleted it and changed the folder to lowercase

**Comment by Justinfungi** [comment]
> @Justinfungi Sorry for the late reply, it seems some other commits introduce a new conflict. I will merge this once you could fix it.

@ashawkey Hah its ok. I think now the conflict is solved

**Comment by ashawkey** [comment]
Thanks for your contribution!

**Pull request merged**